install/kubernetes/generate-webhook-cert.sh

```diff
@@ -0,0 +1,128 @@
+#!/bin/sh
+
+set -e
+
+function usage() {
+    cat <<EOF
+Generate certificate suitable for use with an Istio webhook service.
+
+This script demonstrates how to use the CloudFlare's PKI toolkit
+(cfssl) and k8s' CertificateSigningRequest API to a generate a
+certificate signed by k8s CA suitable for use with Istio webhook
+services. This requires permissions to create and approve CSR. See
+https://kubernetes.io/docs/tasks/tls/managing-tls-in-a-cluster for
+detailed explantion and additional instructions.
+
+The server key/cert k8s CA cert are stored in a k8s secret.
+
+usage: ${0} [OPTIONS]
+
+The following flags are required.
+
+       --service          Service name of webhook.
+       --namespace        Namespace where webhook service resides.
+       --secret           Secret name for CA certificate and server certificate/key pair.
+       --secret-namespace Namespace where secret should be created. Default
+EOF
+    exit 1
+}
+
+while [[ $# -gt 0 ]]; do
```

Comment:
I'm unable to get this to work with the following example
```
./generate-webhook-cert.sh --service test --namespace default --secret webhook --secret-namespace default
```

Reply:
I'm unable to see or reproduce the error. Any additional information on what's not working here?

Reply:
ok I think I must have had a typo. It works with the latest patchset now.